### ISSUE TITLE
Build fails on base-4.8

### ISSUE THREAD

**Issue by pharpend** [david-christiansen/annotated-wl-pprint#8]
I happened upon this while attempting to install Idris:

```
Resolving dependencies...
Configuring annotated-wl-pprint-0.5.3...
Building annotated-wl-pprint-0.5.3...
Failed to install annotated-wl-pprint-0.5.3
Build log ( /home/pete/.cabal/logs/annotated-wl-pprint-0.5.3.log ):
Configuring annotated-wl-pprint-0.5.3...
Building annotated-wl-pprint-0.5.3...
Preprocessing library annotated-wl-pprint-0.5.3...

on the commandline: Warning:
    -XOverlappingInstances is deprecated: instead use per-instance pragmas OVERLAPPING/OVERLAPPABLE/OVERLAPS
[1 of 1] Compiling Text.PrettyPrint.Annotated.Leijen ( Text/PrettyPrint/Annotated/Leijen.hs, dist/build/Text/PrettyPrint/Annotated/Leijen.o )

Text/PrettyPrint/Annotated/Leijen.hs:214:24:
    Ambiguous occurrence ‘<$>’
    It could refer to either ‘Text.PrettyPrint.Annotated.Leijen.<$>’,
                             defined at Text/PrettyPrint/Annotated/Leijen.hs:273:3
                          or ‘Prelude.<$>’,
                             imported from ‘Prelude’ at Text/PrettyPrint/Annotated/Leijen.hs:1:8-40
                             (and originally defined in ‘Data.Functor’)
cabal: Error: some packages failed to install:
annotated-wl-pprint-0.5.3 failed during the building phase. The exception was:
ExitFailure 1
```

**Comment by david-christiansen** [comment]
Version 0.6.0 on Hackage should have solved building with 7.10. But it looks like Idris needs to have its upper bound adjusted.

Thanks! I'll close this and bump the bound in Idris.

**Comment by david-christiansen** [comment]
https://github.com/idris-lang/Idris-dev/pull/2167 should fix this for you.

Thanks again!
